### DISCUSSION TITLE
Does 1980 - 2016 climate maps match with 1950 - 2021 data ?

### DISCUSSION THREAD

**Discussion by SanderDevisscher** [inbo/riparias-prep#19] [Q&A]
We selected Gbif occurrences between 1950 and 2021. Is this period compatible with the climate map you are using (1976-2000 or 1980-2016)?
Would it be necessary to use an earlier climate map for occurrences before 1976/1980?

**Comment by RomainWilleput** [comment]
I found this: http://koeppen-geiger.vu-wien.ac.at/shifts.htm 
We could use the maps 1926-1950, 1951-1975, 1976-2000 and 2001-2025 (several existing scenarios) for the different Gbif occurrences according to the date (from 1950 to 2021).
I think this will make the analysis more accurate than using the 1980-2016 map for all dates.
I don't know if this is possible with the current script.
Also, I see that there are several scenarios (A1FI, A2, B1, B2) for the 2076-2100 map, which one was used in the current analysis?

**Reply by SanderDevisscher** [reply]
@RomainWilleput I think its possible to implement some logic to do more detailed intersects with these scenarios.
But since #17 contains "working" code I would prefer if we merge this into the master and start a new issue to implement this logic. 

Also I can't access the link you provided!

**Reply by SanderDevisscher** [reply]
@RomainWilleput can you provide a working link to the different scenarios?

**Reply by SanderDevisscher** [reply]
> Also, I see that there are several scenarios (A1FI, A2, B1, B2) for the 2076-2100 map, which one was used in the current analysis?

I used the 2071–2100 scenario as described in [beck et al.](https://www.nature.com/articles/sdata2018214)

**Reply by RomainWilleput** [reply]
> @RomainWilleput can you provide a working link to the different scenarios?

For some reason, the link only works from my university's wifi network (but not from my home)
Here are the available datasets:
![IMG_20210510_124718](https://user-images.githubusercontent.com/81576915/117648223-3fdf4000-b18e-11eb-91bb-64fc8a7c6c62.jpg)

If it seems useful, I can try to download them at the university and then send them to you.
